### ISSUE TITLE
Build data processing pipeline v0.1

### ISSUE THREAD

**Issue by barisozmen** [barisozmen/deepaugment#5]
Data preprocess should be like:
1. Remove images having width or height less than 608*
2. Split images using SplitImg.py module of DOTA_devkit, where subsize=608 and gap=0.
3. Remove any image whose after-split dimensions are not order of 608
4. Convert oriented bounding boxes (OBB) to horizontal bounding boxes (HBB)

For pipeline v0.1, only use 20 images for training set, where 10 of them having "planes" in it. All images from test set. MVP targets only to detect planes.

* The rationale of choosing 608 as size is that pre-trained model, which I will use in v0.1, was trained by 608x608 images (https://github.com/ringringyi/DOTA_models#training).
